code/controllers/subsystem/time_track.dm

```diff
@@ -59,6 +59,9 @@ SUBSYSTEM_DEF(time_track)
 			"tidi_avg",
 			"tidi_slowavg",
 			"maptick",
+			"average starting tick usage",
+			"average sleeping tick usage",
+			"average sleeping tick overtime",
```

Comment:
_ seperated please, also avg_start_tick_use maybe? or just sleep_use_avg and sleep_over_avg? less longwinded the better, while still remaining clear and all